azure-pipelines-product-construction-service.yml

```diff
@@ -154,55 +154,79 @@ stages:
+  - stage: DeployPCS
+    dependsOn:
+    - Build
+    displayName: Deploy Product Construction Service
+    
+    jobs:
+    - job: Deploy
+      displayName: Deploy container app
+      pool:
+        name: NetCore1ESPool-Internal
+        demands: ImageOverride -equals 1es-windows-2019
+      variables:
+        newDockerImageTag: $[stageDependencies.Build.BuildAndPublishDocker.outputs['DockerTag.newDockerImageTag']]
 
-          steps:
-          - powershell: |
-              Write-Host $(containerjobNames)
-              $az = Get-Command az.cmd
-              "##vso[task.setvariable variable=azCliPath]$($az.Source)"
+      steps:
+      - powershell: |
+          Write-Host $(containerjobNames)
+          $az = Get-Command az.cmd
+          "##vso[task.setvariable variable=azCliPath]$($az.Source)"
 
-          - powershell: Write-Host $(azCliPath)
+      - powershell: Write-Host $(azCliPath)
 
-          - task: AzureCLI@2
-            inputs:
-              azureSubscription: $(serviceConnectionName)
-              scriptType: pscore
-              scriptLocation: inlineScript
-              inlineScript: |
-                New-Item -ItemType Directory -Path $(diffFolder)
-                $before = az containerapp show --name $(containerappName) -g $(resourceGroupName) --output json
-                Set-Content -Path $(diffFolder)/before.json -Value $before
-            displayName: Snapshot configuration (before)
+      - task: AzureCLI@2
+        inputs:
+          azureSubscription: $(serviceConnectionName)
+          scriptType: pscore
+          scriptLocation: inlineScript
+          inlineScript: |
+            New-Item -ItemType Directory -Path $(diffFolder)
+            $before = az containerapp show --name $(containerappName) -g $(resourceGroupName) --output json
+            Set-Content -Path $(diffFolder)/before.json -Value $before
+        displayName: Snapshot configuration (before)
 
-          - download: current
-            displayName: Download ProductConstructionService.Deployment
-            artifact: ProductConstructionService.Deployment     
+      - download: current
+        displayName: Download ProductConstructionService.Cli
+        artifact: ProductConstructionService.Cli     
 
-          - task: NuGetToolInstaller@1
-            displayName: Use NuGet
-            inputs:
-              versionSpec: 5.3.x
+      - task: NuGetToolInstaller@1
+        displayName: Use NuGet
+        inputs:
+          versionSpec: 5.3.x
 
-          - powershell: |
-              . .\eng\common\tools.ps1
-              InitializeDotNetCli -install:$true
-              .\.dotnet\dotnet workload install aspire
-            displayName: Install .NET and Aspire Workload
+      - powershell: |
+          . .\eng\common\tools.ps1
+          InitializeDotNetCli -install:$true
+          .\.dotnet\dotnet workload install aspire
+        displayName: Install .NET and Aspire Workload
 
+<<<<<<< HEAD
```

Comment:
Something went wrong here

Reply:
huh, I didn't get any conflict message about that file from git, weird